k8s/memcached/apptest/tester/tests/basic-suite.yaml

```diff
@@ -17,3 +17,12 @@ actions:
     expect:
       stdout:
         equals: "STORED\r\nVALUE bar 0 9\r\nsome text\r\nEND\r\nSTORED\r\nVALUE bar 0 9\r\nsome text\r\nEND\r\nSTORED\r\nVALUE bar 0 9\r\nsome text\r\nEND"
+- name: Prometheus metrics are exported
```

Comment:
Optionally - We can use a loop here (written in go template) to have separate actions for each node.

Reply:
Actually, here we have a loop over all pods, not a set number, so a loop with hardcoded number would not work.

Also, it would need to connect to each instance directly, without using a service.